ui/client/admin.js

```diff
@@ -42,6 +43,7 @@ const Admin = () => {
   const [nodeInfo, setNodeInfo] = useState([]);
   const [shutDownFailed, setShutDownFailed] = useState(false);
   const [alertMessage, setAlertMessage] = useState('');
+  usePageTitle({ title: 'Admin' });
```

Comment:
This is the standard example of the change made to most of the files in this PR: adding this hook, removing the useEffect.